tests/linalg_test.py

```diff
@@ -804,21 +804,29 @@ def args_maker():
     self._CompileAndCheck(jnp.linalg.inv, args_maker)
 
   @jtu.sample_product(
-    shape=[(1, 1), (4, 4), (2, 70, 7), (2000, 7), (7, 1000), (70, 7, 2),
-           (2, 0, 0), (3, 0, 2), (1, 0)],
+    [dict(shape=shape, hermitian=hermitian)
+     for shape in [(1, 1), (4, 4), (3, 10, 10), (2, 70, 7), (2000, 7),
+                   (7, 1000), (70, 7, 2), (2, 0, 0), (3, 0, 2), (1, 0)]
+     for hermitian in ([False, True] if shape[-1] == shape[-2] else [False])],
     dtype=float_types + complex_types,
   )
   @jtu.skip_on_devices("rocm")  # will be fixed in ROCm-5.1
-  def testPinv(self, shape, dtype):
+  def testPinv(self, shape, hermitian, dtype):
     rng = jtu.rand_default(self.rng())
-    args_maker = lambda: [rng(shape, dtype)]
+    if hermitian:
+      def args_maker():
+        x = rng(shape, dtype, scale=1)
+        return [np.matmul(x, T(x.conj())) + np.eye(shape[-1], dtype=dtype)]
```

Comment:
Since input is symmetrized, is it worth checking with non-hermitian input (e.g. to make sure the gradient is computing the right thing)?

Reply:
Done. (I actually had to add additional symmetrization in the JVP as well.)